.github/mergify.yml

```
@@ -16,11 +18,13 @@ queue_rules:
       - check-success=Clippy
       - check-success=Rustfmt
 
-  - name: medium
+  - name: high
     allow_inplace_checks: False
-    allow_checks_interruption: False
-    speculative_checks: 2
-    batch_size: 3
+    allow_checks_interruption: True
+    speculative_checks: 3
```

This doesn't speed things up, because parallel speculative checks are re-run when the first batch merges.

```suggestion
    speculative_checks: 1
```